3-networks-dual-svpc/modules/restricted_shared_vpc/dns.tf

```diff
@@ -32,14 +32,19 @@ resource "google_dns_policy" "default_policy" {
  Creates DNS Peering to DNS HUB
 *****************************************/
 data "google_compute_network" "vpc_dns_hub" {
-  name    = "vpc-net-dns"
-  project = var.dns_hub_project_id
+
+  count = var.environment_code != "p" ? 1 : 0
+
+  name    = "vpc-p-shared-restricted"
```

Comment:
not sure if `"vpc-p-shared-restricted"` should be hardcoded, if the end-user change naming conventions this would fail

Reply:
I believe it is fine to hardcode this, as the name is a composition of the locals in the [main.tf](https://github.com/renato-rudnicki/terraform-example-foundation/blob/e9867b6f3a5976436459461a05e9f30b4c55e56c/3-networks-dual-svpc/modules/restricted_shared_vpc/main.tf#L19) file. Additionally, some hardcoded content is included in the modules to ensure that the basic structure is created as intended and we don't recommend the customer to change it. Usually, we only allow the customer to modify values in the examples / envs."